src/Generator.php

```diff
@@ -119,7 +119,7 @@ public function __construct(array $settings = null)
 	 */
 	public function setSecure($secure = true)
 	{
-		$this->secure = (bool) $secure;
+		$this->secure = (bool)$secure;
```

Comment:
I'm not sure, I cheated by running PHPStorm's auto styler :wink:, but my config isn't quite right (see the Circle CI output).